### PR TITLE
Add quark gluon likelihood to MiniAOD

### PR DESCRIPTION
**Pull request by ahinzmann**
This PR adds the userFloat "QGTagger:qgLikelihood" to slimmedJets, a variable to distinguish quark and gluon jets described in JME-16-003.
It was checked that userFloat is filled correctly with reasonable values, when running the MiniAOD sequence.
Size and timing with/without this change is summarized below

WITHOUT QGTagger:qgLikelihood:
patJets_slimmedJets__PAT. 27740.8 2602.11
TimeReport> Time report complete in 2488.15 seconds
Time Summary:

    Min event: 0.646415
    Max event: 30.6585
    Avg event: 2.42614
    Total loop: 2454.44
    Total job: 2488.15
    Event Throughput: 0.407426 ev/s
    CPU Summary:
    Total loop: 1875.75
    Total job: 1898.77

WITH QGTagger:qgLikelihood:
patJets_slimmedJets__PAT. 28162.9 2653.54
TimeReport> Time report complete in 2502.59 seconds
Time Summary:

    Min event: 0.658903
    Max event: 29.1476
    Avg event: 2.44299
    Total loop: 2471.61
    Total job: 2502.59
    Event Throughput: 0.404595 ev/s
    CPU Summary:
    Total loop: 1873.68
    Total job: 1897.58
    TimeReport ---------- Module Summary ---[Real sec]----
    TimeReport per event per exec per visit Name
    TimeReport 0.001683 0.001683 0.001683 QGTagger

### CODE REVIEW

**Comment by slava77** [comment]
@cmsbuild please test

File: PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py [hunk 1]
```diff
@@ -182,6 +182,10 @@ def miniAOD_customizeCommon(process):
     process.patJets.userData.userFloats.src = [ cms.InputTag("pileupJetId:fullDiscriminant"), ]
     process.patJets.userData.userInts.src = [ cms.InputTag("pileupJetId:fullId"), ]
 
+    ## Quark Gluon Likelihood
+    process.load('RecoJets.JetProducers.QGTagger_cfi')
+    process.patJets.userData.userFloats.src += ['QGTagger:qgLikelihood']
+
```
Comment (gpetruc) [inline PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py]: perhaps the item needs also to be added to a cms.Task?

Comment (ahinzmann) [inline PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py]: Indeed. Now the miniaod step runs fine.

**Comment by perrotta** [comment]
please test

**Comment by slava77** [comment]
+1

for https://github.com/cms-sw/cmssw/pull/18307	ea14319
- adds QGtagger to miniAOD jet userFloats
    - Note that QGtagger runs in standard relval workflows as a part of DQM. So, there are no technical issues with it. The CPU in ttbar PU35 2016 setup for this module is about 1 ms/evt and all needed inputs are available in AOD to run by itself.
- jenkins test pass  and comparisons show no relevant differences

**Pull request merged**